PythonExample/main.py

```diff
@@ -0,0 +1,62 @@
+from lockstep.lockstep_api import LockstepApi
+import os
+
+
+def retrieve_api_key():
+    API_KEY = os.environ.get('LOCKSTEP_API_SBX')
+    if API_KEY is None:
+        print('NO API KEY')
+    else:
+        print('VALID API KEY FOUND')
+        return API_KEY
+
+
+def create_client(apikey):
+    #     create client
+    env = 'https://api.sbx.lockstep.io/'
+    client = LockstepApi(env)
```

Comment:
Suggestion to simplify the code: 

```
client = LockstepApi("sbx")
```

We want to encourage our customers to use the simplest possible approach and not ask them to remember to type in the complete URL each time.

Reply:
This is done